easybuild/easyconfigs/n/ncurses/ncurses-6.0.eb

```diff
@@ -16,9 +16,10 @@ sources = [SOURCE_TAR_GZ]
 
 configopts = [
     # default build
-    '--with-shared --enable-overwrite',
+    '--with-shared --enable-overwrite --with-termlib',
```

Comment:
@ocaisa this should be reflected in the `sanity_check_paths` too?

Reply:
See #5067